Fix build failure with gcc 4.2.1:
gkqueuefilemonitor.c:88: error: redefinition of typedef 'GKqueueFileMonitor'
kqueue-helper.h:31: error: previous declaration of 'GKqueueFileMonitor' was here
https://gitlab.gnome.org/GNOME/glib/issues/1506
https://gitlab.gnome.org/GNOME/glib/merge_requests/291
--- gio/kqueue/gkqueuefilemonitor.c.orig	2018-08-30 12:12:08.000000000 -0500
+++ gio/kqueue/gkqueuefilemonitor.c	2018-08-31 09:42:20.000000000 -0500
@@ -75,7 +75,7 @@
  *
  * To distinguish between a directory monitor and a regular file monitor, check
  * whether sub_file is NULL. */
-typedef struct _GKqueueFileMonitor
+struct _GKqueueFileMonitor
 {
   GLocalFileMonitor parent_instance;
 
@@ -85,7 +85,7 @@
   GFileMonitor *fallback;
   GFile *fbfile;
 #endif
-} GKqueueFileMonitor;
+};
 
 #ifdef __clang__
 #pragma clang diagnostic pop
